src/types/transaction.ts

```diff
@@ -108,28 +108,30 @@ export type TransactionContextType = {
 
 export type ContextSummaryVariableType =
   | {
-      type:
-        | 'emphasis'
-        | 'address'
-        | 'transaction'
-        | 'eth'
-        | 'farcasterID'
-        | 'crosschain';
+      type: 'address' | 'transaction' | 'eth' | 'farcasterID' | 'crosschain';
```

Comment:
@ponyjackal The fastest path to fixing the build errors is to allow for 'string' here, too, and then convert all the `emphasis` types to
```
{
  type: 'string',
  emphasis: true
}
```suggestion
      type: 'address' | 'transaction' | 'eth' | 'farcasterID' | 'crosschain' | 'string';
```